intuita-webview/src/CreateIssue/index.tsx

```diff
@@ -78,8 +88,11 @@ const CreateIssue = (props: Props) => {
 	}, [props.title]);
 
 	useEffect(() => {
+		if (props.loading) {
+			return;
+		}
 		editor?.commands.setContent(props.body, false);
-	}, [editor, props.body]);
+	}, [editor, props.body, props.loading]);
```

Comment:
is this a bug fix?

Reply:
yes. this is needed to do the following check
```
if (props.loading) { return; }
```

which is needed to prevent the content changing when user presses "create issue" button. the content will change if `convertHTMLCodeBlockToMarkdownString` results in any change.